File: UI/rank-oh-meter/src/main/java/ro/unibuc/rankohmeter/models/LolModel.java
package ro.unibuc.rankohmeter.models;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
public class LolModel {

    private Long id;

    private String name;

    private Long wins;

    private Long losses;

    private String division;

    private Long points;

    private String most_used_champs;

    private Long kills;

    private Long deaths;

    private Long assists;
}

File: UI/frontend/src/app/models/lol.model.ts
export class LolModel {
  constructor(public id: number,
              public name: string,
              public wins: number,
              public losses: number,
              public division: string,
              public points: number,
              public mostUsedChamps: string,
              public kills: string,
              public deaths: string,
              public assists: string,
  ) {
  }
}
